.github/workflows/wrapper.yml

```diff
@@ -0,0 +1,10 @@
+name: Validate Gradle Wrapper
+on: [push, pull_request]
+
+jobs:
+  validate:
+    name: Validate
+    runs-on: ubuntu-latest
```

Comment:
Do we support windows builds on OpenSearch? 
If so we should run these checks on multi platforms.

Reply:
Let's do other platforms separately.